gci/management/commands/fetch_deployed_data.py

```diff
@@ -13,12 +13,15 @@ class Command(BaseCommand):
 
     def add_arguments(self, parser):
         parser.add_argument('output_dir', nargs='?', type=str)
-        parser.add_argument('filenames', nargs='+', type=str)
+        parser.add_argument('filenames', nargs='?', type=str)
+        parser.add_argument('allow_failure', nargs='?', type=bool,
+                            default=False)
 
     def handle(self, *args, **options):
         logger = logging.getLogger(__name__)
         output_dir = options.get('output_dir')
-        filenames = options.get('filenames')
+        filenames = options.get('filenames').split(' ')
```

Comment:
Im doing https://github.com/coala/community/issues/132 ..

But .. this is not desirable

It is better to let them be separate args.

If you want to use `allow_failure`, call it on the cli like `--allow_failure`